packages/react-composites/tests/browser/call/PageState.test.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { expect } from '@playwright/test';
+import { clickOutsideOfPage, waitForPageFontsLoaded, waitForSelector, dataUiId } from '../common/utils';
+import { test } from './fixture';
+import { buildUrlWithMockAdapter } from './utils';
+
+test.describe('Page state tests', async () => {
+  test('Page when waiting in lobby', async ({ pages, serverUrl }) => {
+    const page = pages[0];
+    await page.goto(
```

Comment:
nit: there's a lot of duplicate code per test that isn't really related to the test itself (and would need updated everywhere if we needed to add a new condition/updated a fn name), consider having a helper function:
```ts
gotoPageWithMockAdapter = (qArgs) => {
  await page.goto(..);
  await clickOutsideOfPage(page);
  await waitForPageFontsLoaded(page);
}
```

Reply:
Since you have multiple PRs in flight at add tests. I'd recommend landing these and write a PR that extracts this out from all the `mockAdapter` tests.